### PR TITLE
Editorial changes

### PR DESCRIPTION
**Pull request by hawkinsw**
Just fixing a few typos and proposing some alternate wording in places.

### CODE REVIEW

**Comment by hawkinsw** [comment]
@marten-seemann Thank you for a great proposal. I just saw a few things that I thought maybe I could help fix. I hope it is helpful -- please feel free to ignore if it's not useful!

File: draft-seemann-quic-reliable-stream-reset.md [hunk 1]
```diff
@@ -94,18 +95,18 @@ RELIABLE_RESET_STREAM Frame {
 
 RELIABLE_RESET_STREAM frames contain the following fields:
 
-Stream ID:  A variable-length integer encoding of the stream ID of
+Stream ID:  A variable-length-encoded integer encoding of the stream ID of
```
Comment (marten-seemann) [inline draft-seemann-quic-reliable-stream-reset.md:98]: This was just copied from the RFC, so I'd prefer to keep it as is: https://datatracker.ietf.org/doc/html/rfc9000#section-19.4

```suggestion
Stream ID:  A variable-length integer encoding of the stream ID of
```

File: draft-seemann-quic-reliable-stream-reset.md [hunk 2]
```diff
@@ -94,18 +95,18 @@ RELIABLE_RESET_STREAM Frame {
...
       the stream being terminated.
 
-Application Protocol Error Code:  A variable-length integer
+Application Protocol Error Code:  A variable-length-encoded integer
```
Comment (marten-seemann) [inline draft-seemann-quic-reliable-stream-reset.md:101]:
```suggestion
Application Protocol Error Code:  A variable-length integer
```

File: draft-seemann-quic-reliable-stream-reset.md [hunk 3]
```diff
@@ -94,18 +95,18 @@ RELIABLE_RESET_STREAM Frame {
...
     containing the application protocol error code (see Section 20.2)
     that indicates why the stream is being closed.
 
-Final Size:  A variable-length integer indicating the final size of
+Final Size:  A variable-length-encoded integer indicating the final size of
```
Comment (marten-seemann) [inline draft-seemann-quic-reliable-stream-reset.md:105]:
```suggestion
Final Size:  A variable-length integer indicating the final size of
```

File: draft-seemann-quic-reliable-stream-reset.md [hunk 4]
```diff
@@ -94,18 +95,18 @@ RELIABLE_RESET_STREAM Frame {
...
     the stream by the RESET_STREAM sender, in units of bytes; see
     (Section 4.5 of {{!RFC9000}}).
 
-Reliable Size:  A variable-length integer indicating the amount of
+Reliable Size:  A variable-length-encoded integer indicating the amount of
```
Comment (marten-seemann) [inline draft-seemann-quic-reliable-stream-reset.md:109]:
```suggestion
Reliable Size:  A variable-length integer indicating the amount of
```

**Review by marten-seemann — APPROVED** [review]
Thank you @hawkinsw!

**Review by marten-seemann — APPROVED** [review]

**Pull request merged**